src/components/AlignContentPicker/index.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+
+import { ToggleGroup, ToggleGroupItem } from '@/components/ui/toggle-group'
+import { cn } from '@/utilities/ui'
+import { FieldLabel, useField } from '@payloadcms/ui'
+import { AlignCenter, AlignLeft, AlignRight } from 'lucide-react'
+import { SelectFieldClientProps } from 'payload'
+
+const AlignContentPicker = (props: SelectFieldClientProps) => {
+  const alignOptions = [
+    { value: 'left', label: 'Left', icon: AlignLeft },
+    { value: 'center', label: 'Center', icon: AlignCenter },
+    { value: 'right', label: 'Right', icon: AlignRight },
+  ]
+
+  const { path, field } = props
+
+  const { value, setValue } = useField({ path })
+
+  const currentValue = (value as string) || 'left'
```

Comment:
Do we need this type casting here? what if we set the type for value above as string?